src/test/java/com/assignmentTest/Assignment3Test.java

```diff
@@ -0,0 +1,20 @@
+package com.assignmentTest;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import com.assignment.Assignment3;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+public class Assignment3Test {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    @Test
+    public void getCharacterIndex(){
+    Assignment3 indexCharacter=new Assignment3();
```

Comment:
It is a good practice to have the object name = class name but in lower case.
Eg : `Assignment3 assignment3=new Assignment3()`